modules/private-clusters-setting-api-private.adoc

```diff
@@ -31,8 +31,58 @@ you can reconfigure the API server to use only the private zone.
 
 .Procedure
 
+. In the web portal or console for your cloud provider, take the following actions:
+
+.. Locate and delete the appropriate load balancer component:
+ifndef::cpmso-using-azure[]
+*** {aws-short} clusters: Delete the external load balancer. The API DNS entry in the private zone already points to the internal load balancer, which uses an identical configuration, so you do not need to modify the internal load balancer.
+endif::cpmso-using-azure[]
+ifndef::cpmso-using-aws[]
+*** {azure-short}: Delete the following resources:
+ ** The `api-v4` rule for the public load balancer.
+ ** The `frontendIPConfiguration` parameter that is associated with the `api-v4` rule for the public load balancer.
+ ** The public IP that is specified in the `frontendIPConfiguration` parameter.
+
+.. {azure-short} clusters: Configure the Ingress Controller endpoint publishing scope to `Internal`.
+For more information, see "Configuring the Ingress Controller endpoint publishing scope to Internal".
++
+[IMPORTANT]
+====
+If you configure the Ingress Controller endpoint publishing scope to `Internal` and there are no existing inbound rules in the public load balancer, you must create an outbound rule explicitly to provide outbound traffic for the backend address pool.
```

Comment:
Starting from 4.17+, installer create the outbound rule explicitly in public load balancer to provide outbound traffic, so I think there is no need to add this IMPORTANT admonition.